### DISCUSSION TITLE
Pull to search

### DISCUSSION THREAD

**Discussion by Sohailwahab** [wstyres/Zebra#630] [Ideas]
It’s simply a feature that should’ve been everywhere within iOS that contains a search bar on top of the UI. 

FEATURE REQUEST: Pull down in the middle of the search interface to focus on search and bring up they keyboard.

**Comment by jakeva** [comment]
Search should also be available when inside a repository.

**Comment by worldlydev** [comment]
> Search should also be available when inside a repository.

Came here to say this since my issue was closed in favor of this one. There should be pull to search from within individual repos and from the packages tab (such as Sileo).

----

Also a minor update. 

I have a suggestion for the "Search for packages" string. I think you should follow Apple's design and change the search text to something that gives multiple examples of what you can search for. Such as in the app store which says you can search for "games, apps, articles, and more". I feel that "Search for packages" should be changed to "Tweaks, themes, widgets, and more"or something similar. 

* Pull to search in repos and packages tab. For ease of use reasons. Easy to search within specific repos and through installed packages.
* Change search string. Make it more consistent with iOS. Remove "Search for" as it's a search bar. Plus include examples of what can be searched for instead of just packages.

![D7DBA048-FAEE-4806-883D-589DDAFCA6AD](https://user-images.githubusercontent.com/51668635/71192848-54652480-2289-11ea-9564-8f493cc29e47.jpeg)
